src/main/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -1663,8 +1663,11 @@ public String getContig() {
     }
 
     /**
-     * Returns 1-based inclusive start position of the variant, 0 or greater.
-     * See below for explanation on "0".
+     * Returns 1-based inclusive start position of the variant.
```

Comment:
I would drop the "start" since this is in fact controversial, despite that it is the method's name.

Reply:
Also would people know what "1-based inclusive" mean? perhaps that should be remove from here and or explained in another "<p/>" block... but since this not the only place  we use 1-based indexes ... so do you even need to explain it here.....  I guess it won't hurt.

Reply:
> I would drop the "start" since this is in fact controversial, despite that it is the method's name.

Why is "start" controversial?

I don't quite understand the rest of the comments